apps/src/StudioApp.js

```diff
@@ -2890,6 +2890,7 @@ StudioApp.prototype.setPageConstants = function (config, appSpecificConstants) {
     puzzleNumber: level.puzzle_number,
     stageTotal: level.stage_total,
     noVisualization: false,
+    visualizationInWorkspace: false,
```

Comment:
This new flag means that the "running app" (aka visualization) is actually inside the workspace (typically, it's not - it is to the left of the workspace)